File: plugins/kotlin/idea/tests/testData/findUsages/libraryUsages/findLibraryFunctionUsages/kotlinTopLevelMethodUsagesMultifile/kotlinTopLevelMethodUsagesMultifile.0.java
// PSI_ELEMENT: org.jetbrains.kotlin.analysis.decompiled.light.classes.KtLightMethodForDecompiledDeclaration
// OPTIONS: usages
// PSI_ELEMENT_AS_TITLE: "fun processRequest(): String"
package client;

import server.RequestProcessor;

class JClient {
    String s = RequestProcessor.<caret>processRequest();

    String sendRequest() {
        return RequestProcessor.processRequest();
    }
}

File: plugins/kotlin/idea/tests/testData/findUsages/kotlin/findFunctionUsages/kotlinTopLevelMethodUsagesMultifile.0.kt
// PSI_ELEMENT: org.jetbrains.kotlin.psi.KtNamedFunction
// OPTIONS: usages
// PSI_ELEMENT_AS_TITLE: "fun processRequest()"
@file:[JvmName("RequestProcessor") JvmMultifileClass]

package server

fun <caret>processRequest() = "foo"

